tests/unit/test_backends.py

```diff
@@ -190,24 +190,24 @@ def toProtocolElement(self):
                 return self
 
         self.request = FakeRequest()
-        self.request.pageToken = None
-        self.numObjects = 3
-        self.objects = [FakeTopLevelObject() for j in range(self.numObjects)]
+        self.request.page_token = None
```

Comment:
Yes it should, I was wondering how does this work (as protobuf doesn't allow None assignments and that'd normally be an error), but here we're using a FakeRequest object instead. I'll update it.